jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-commit.yaml

```diff
@@ -0,0 +1,135 @@
+# Bootstrap jobs that trigger on commits
+
+- job-template:
+    builders:
+    - shell: |
+        # TODO(fejta): consider a stable tag instead of master
+        git clone https://github.com/kubernetes/test-infra -b master
+        './test-infra/jenkins/bootstrap.py' --job='{job-name}' --repo='{repo-name}' --branch='{branch}' --root="${{GOPATH}}/src"
+    disabled: false
+    name: 'ci-{commit-suffix}'
+    node: 'build'
+    properties:
+    - build-discarder:
+        num-to-keep: 200
+    scm:  # TODO(fejta): migrate scm monitoring to prow.
+    - git:
+        url: '{giturl}'
+        basedir: '${{GOPATH}}/src/{repo-name}'
+        branches:
+        - '{branch}'
+        skip-tag: true
+        wipe-workspace: false
+    triggers:
+    - pollscm:
+        cron: '{commit-frequency}'
+    - timed: '{timed-frequency}'
```

Comment:
Why is there a timed frequency here? None of the jobs you're migrating from had one.